src/services/townServices.ts

```diff
@@ -0,0 +1,14 @@
+import { Town } from "@/models/town";
+import axiosInstance from '../configs/apiConfig';
+import { TOWN_ENDPOINT } from "@/configs/endpoints";
+
+export class TownService{
+
+    create(payload: Town){
+      return  axiosInstance.post(TOWN_ENDPOINT+'/create',payload)
```

Comment:
if create is a const you can add it in enpoints file export it anywhere then you can avoid redundant endpoint